src/hydration/bottles.ts

```diff
@@ -88,14 +95,18 @@ because those items would otherwise become normal bottles without custom model d
 // Fill a bottle
 registerEvent(PlayerInteractEvent, async (event) => {
   if (event.item?.type !== Material.GLASS_BOTTLE) return;
-  if (!event.item.itemMeta.hasCustomModelData()) return; // Default glass bottle
-  const a = event.action;
-  if (a !== Action.RIGHT_CLICK_AIR && a !== Action.RIGHT_CLICK_BLOCK) return;
+  // if (!event.item.itemMeta.hasCustomModelData()) {
+  //   // Default glass bottle
+
+  //   return;
+  // }
+  if (!isRightClick(event.action)) return;
 
   // Didn't compile? Could this be used instead of setCancelled?
-  //event.setUseItemInHand(Result.DENY)
+  //event.setUseItemInHand(Result.DENY);
```

Comment:
Please re-confirm: does this *still* not compile?

Reply:
Compiles, but cancelling the event seems like the best approach. Comments will be deleted now